AUTHORS

```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
 Marco Antônio Habitzreuter <mahabitzreuter@gmail.com>
 Ayush Bisht <bisht.ayush2001@gmail.com>
 Akshansh Bhatt <akshansh@tuta.io>
+Brandon T. Willard <brandonwillard@users.noreply.github.com>
+Thomas Aarholt <thomasaarholt@gmail.com>
+Hiren Chalodiya <hirenchalodiya99@gmail.com>
+Roland Dixon <rols121@gmail.com>
+dimasvq <dimas.vq.2020@bristol.ac.uk>
+Sagar231 <sagarfeb298@gmail.com>
+Michael Chu <michael02chu@gmail.com>
+Abby Ng <abigailjng@gmail.com>
+Angad Sandhu <55819847+angadsinghsandhu@users.noreply.github.com>
+Alexander Cockburn <alexander_cockburn12@hotmail.com>
+Yaser <yaseralosh@outlook.com>
+Davide_sd <sandona.davide@gmail.com>
+gutow <gutow@uwosh.edu>
+Nihir Agarwal <f20180701@pilani.bits-pilani.ac.in>
+Lee Johnston <lee.johnston.100@gmail.com>
+Zach Carmichael <20629897+craymichael@users.noreply.github.com>
+vijair <vijair@umich.edu>
+vijairam20 <vijairamg@gmail.com>
+dsaw <devsaw115@gmail.com>
+Hanspeter Schmid <hanspeter.schmid@fhnw.ch>
+oostben <oostben@umich.edu>
+Nikita <nikita.student.cse19@itbhu.ac.in>
+Aman <amanmourya295@gmail.com>
+Shashank KS <shashankks@sahaj.ai>
```

Comment:
I've emailed asking what name to use

Reply:
This email gave address not found...

Reply:
@shashankks0987 this entry in the authors file seems to come from your commits.

I tried emailing the address but it doesn't seem to be a valid email address.

What name and email address would you like to have recorded for your contributions?

Currently the name is listed as "Shashank KS" but I can change this to list your full name if you prefer.

Reply:
@oscarbenjamin
```
 Name: Shashank KS ,
 Email: shashankks0987@gmail.com 
```
I'd like this email address and name to be associated with my contributions

Reply:
Thanks, I've added that.